integrations/weaviate/tests/test_document_store.py

```diff
@@ -1,5 +1,6 @@
 from unittest.mock import patch
 
+from haystack_integrations.document_stores.weaviate.document_store import WeaviateDocumentStore
```

Comment:
This import is correct but in examples or docs I would just use `from haystack_integrations.document_stores.weaviate import WeaviateDocumentStore` instead.

Reply:
Will do 👍